### ISSUE TITLE
MkGrizzlyContainer takes 1s to start/close

### ISSUE THREAD

**Issue by andrejserafim** [jcabi/jcabi-http#145]
Can be reproed with 

```java
    public static void main(String[] args) throws IOException {
        for (int i = 0; i < 20000; i++) {
            new MkGrizzlyContainer().start().close();
        }
    }
```

Profiling reveals that it's waiting for 1s at `com.sun.grizzly.Controller#waitUntilSelectorHandlersStop`. 

Looks like the `Controller` is waiting on this AtomicLong before firing the stop events, but the selector handler threads are awaiting the stop events fired *after* we wait here. So it looks like the there's effectively a 1s sleep here.

Anything we can do about this in jcabi?

**Comment by dmarkov** [comment]
@yegor256 dispatch this issue please, see [par.21](http://at.teamed.io/policy.html#21)

**Comment by yegor256** [comment]
@andrejserafim sure, we have to fix this. Maybe you can contribute with a pull request?

**Comment by andreoss** [comment]
@andrejserafim Closing in favor of #126, since there is a huge performance boost between this two versions.

**Comment by 0crat** [comment]
Job `gh:jcabi/jcabi-http#145` is not assigned, can't get performer
